### PR TITLE
Rk 366 edits del option2

### PR DESCRIPTION
**Pull request by Robert-Krajcik**

### CODE REVIEW

**Comment by AARON-CLARK** [comment]
Hi @Robert-Krajcik, you'll want to pull in the recent changes from `dev` regarding changes to the `renv.lock` file. We made some changes in #398 that should remove the `riskassessment` pkg and install it in the GHA workflows.

File: renv.lock [hunk 1]
```diff
@@ -2458,10 +2450,10 @@
     },
     "xfun": {
       "Package": "xfun",
-      "Version": "0.36",
+      "Version": "0.37",
```
Comment (AARON-CLARK) [inline renv.lock:2453]: FYI - I found that updating this to `xfun v0.37` produces an error/warning when downloading reports.

Comment (Robert-Krajcik) [reply renv.lock:2419]: Huh, thanks for letting me know about xfun

**Comment by Robert-Krajcik** [comment]
Ready for review, @AARON-CLARK

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]
Hi @Robert-Krajcik, this looks really good!

I'm wondering if you could please add a `introJS` guide for the new UI elements that @Jeff-Thompson12 created regarding Decision Automation on the upload tab since that got merged into `dev` yesterday? Note that the the gear (⚙️) will display depending on admin status, but I don't think we need to specifically mention the gear. Maybe some general guidance for the whole section/div would be appropriate and say something like this: "Decision automation. Automatically assign pkg risk decisions based on `riskmetric`-derived risk scores when uploading new pkgs to the database."

image

**Comment by Robert-Krajcik** [comment]
Hi @AARON-CLARK 

Can we put ` add a introJS guide` into a separate PR?
This one has already gone way beyond what it was originally intended to do.

**Review by AARON-CLARK — APPROVED** [review]
Hi @Robert-Krajcik, sure! Please open another PR for the decision automation JS. Thanks!

**Pull request merged**